packages/runtime/src/runtimes/react/controls/style.ts

```diff
@@ -63,6 +65,14 @@ function useStyleControlCssObject(
           borderBottomRightRadius: borderRadiusToString(borderRadius?.borderBottomRightRadius) ?? 0,
           borderBottomLeftRadius: borderRadiusToString(borderRadius?.borderBottomLeftRadius) ?? 0,
         }),
+        ...(properties.includes(StyleControlProperty.TextStyle) && {
+          fontFamily: textStyle?.fontFamily ?? '',
```

Comment:
Wouldn't it be better to default to `null` here so that the style is simply not added and it falls back to the default CSS?

Reply:
I need to find official specification for this, but when I'm playing with browser, checking on any unstyled element will return empty string. For example:
<img width="144" alt="image" src="https://user-images.githubusercontent.com/13066728/180967236-409d458a-db66-4123-9018-e4b457795f06.png">

So it might be that the default value for style with type string is an empty string.

This is what the dev tools show on a component without any TextStyle value:
<img width="293" alt="image" src="https://user-images.githubusercontent.com/13066728/180967816-1f20bc03-7aff-499f-8dbb-3e4dea427804.png">

Reply:
Yeah, but generally with CSS in JS a null means you’re not setting it, that way CSS can just inherit. We don’t want to force the values to be empty and instead want it to properly inherit in case there’s a snippet or something else applying a font, no?

Reply:
Okay cool updated!